back/spec/requests/rack_attack_spec.rb

```diff
@@ -342,6 +342,53 @@
     end
   end
 
+  it 'limits confirmation requests from same user to 10 in 24 hours' do
```

Comment:
Perhaps it could be useful to also test that other users can still use the endpoint when it's throttled for other users, but not blocking.